lua/entities/marums_hoverball/shared.lua

```diff
@@ -1,12 +1,16 @@
 ENT.Type = "anim"
-ENT.Base = "base_gmodentity"
+if WireLib then
+    ENT.Base = "base_wire_entity"
+else
+    ENT.Base = "base_gmodentity"
+end
  
-ENT.Category = "Marums pack"
+ENT.Category = "Wired Stuff"
 
-ENT.PrintName= "Marums Hoverball"
-ENT.Author= "Marum"
-ENT.Contact= "don`t"
-ENT.Purpose= "Annoy"
-ENT.Instructions= "Hovers"
+ENT.PrintName= "Wire Marums Hoverball"
+ENT.Author= "Marum Logica CFC"
+ENT.Contact= "Logica Steam : https://steamcommunity.com/id/GetLogical/"
```

Comment:
i don't think your contact info should be on here

Reply:
Remember workshop... yes sure ill remove it on this version